build.sbt

```diff
@@ -1109,7 +1112,8 @@ def scriptedSettings: Seq[Setting[_]] = ScriptedPlugin.scriptedSettings ++
   )
 
 def archetypeVariables(lagomVersion: String) = Map(
-  "LAGOM-VERSION" -> lagomVersion
+  "LAGOM-VERSION" -> lagomVersion,
+  "SCALA-BINARY-VERSION" -> "2.12"
```

Comment:
What's the benefit of making this a variable rather than hardcoding in the `pom.xml` directly?

Reply:
Instead, I think it should prompt for the version when you create a project from the artifact (with 2.12 as the default).